web/src/components/AppBanner.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <v-banner
+    app
+    color="blue"
+    dark
+    icon="mdi-information"
+    class="ma-4"
+  >
+    <span class="title">Anouncement: </span>The NMDC will be replacing legacy identifiers with
```

Comment:
There is a typo in "Anouncement":

```diff
- Anouncement
+ Announcement
```

Reply:
Also, the `title` has a different line height than the text that follows it, resulting in inconsistent line heights in the banner.

<img width="783" alt="image" src="https://github.com/microbiomedata/nmdc-server/assets/134325062/fe625cf9-a5d6-4d17-b25e-ec8d39af8754">

Proposal:
- On _this_ `title` element, set `line-height: inherit;` so that it uses the same line height as the text that follows it
  <img width="783" alt="image" src="https://github.com/microbiomedata/nmdc-server/assets/134325062/89315ea8-84a7-46da-96d9-b6f1b54932d5">

Reply:
Both things are fixed in https://github.com/microbiomedata/nmdc-server/pull/1216